dbt/loader.py

```diff
@@ -121,6 +126,35 @@ def load_project(cls, root_project, all_projects, project, project_name,
 
 
 class SchemaTestLoader(ResourceLoader):
+    @classmethod
+    def load_all(cls, root_project, all_projects, macros=None):
+        tests = {}
+        patches = {}
+        for project_name, project in all_projects.items():
+            project_tests, project_patches = cls.load_project(
+                root_project, all_projects, project, project_name, macros
+            )
+            for unique_id, test in project_tests.items():
+                if unique_id in tests:
```

Comment:
this means that a model can't be tested/described in two different packages, right? I think this currently works in the v1 flow. The original rationale was (I think) that you should be able to add _extra_ tests to models in your dependencies. I can also imagine that it would be good to be able to override the docs for a model/column defined in a package. Thinking through this, I'm pretty ok with dropping support for this pattern, but wanted to raise that the behavior is changing

Reply:
It means that all your tests must be in one and all your descriptions must be in another, at the most (of course you can have both in one file). The alternative is to have some sort of last-write-wins or similar, which seems far worse. Right?